frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -828,39 +835,61 @@ function ReaderFooter:addToMainMenu(menu_items)
-                        width = Screen:getWidth() * 0.6,
-                        value = font_size,
-                        value_min = 8,
-                        value_max = 36,
-                        default_value = 14,
-                        ok_text = _("Set size"),
-                        title_text =  _("Footer font size"),
-                        callback = function(spin)
-                            self.settings.text_font_size = spin.value
-                            local text = self.footer_text.text
-                            self.footer_text = TextWidget:new{
-                                text = text,
-                                face = Font:getFace(self.text_font_face, self.settings.text_font_size)
+                sub_item_table = {
+                    {
+                        text_func = function()
+                            return T(_("Font size (%1)"), self.settings.text_font_size)
+                        end,
+                        callback = function(touchmenu_instance)
+                            local SpinWidget = require("ui/widget/spinwidget")
+                            local font_size = self.settings.text_font_size
+                            local items_font = SpinWidget:new{
+                                width = Screen:getWidth() * 0.6,
+                                value = font_size,
+                                value_min = 8,
+                                value_max = 36,
+                                default_value = 14,
+                                ok_text = _("Set size"),
+                                title_text =  _("Footer font size"),
+                                callback = function(spin)
+                                    self.settings.text_font_size = spin.value
+                                    self.footer_text:free()
+                                    self.footer_text = TextWidget:new{
+                                        text = self.footer_text.text,
+                                        face = Font:getFace(self.text_font_face, self.settings.text_font_size),
+                                        bold = self.settings.text_font_bold,
+                                    }
+                                    self.text_container[1] = self.footer_text
+                                    self:refreshFooter(true, true)
+                                    if touchmenu_instance then touchmenu_instance:updateItems() end
+                                end,
                             }
-                            self.text_container = RightContainer:new{
-                                dimen = Geom:new{ w = 0, h = self.height },
-                                self.footer_text,
+                            UIManager:show(items_font)
+                        end,
+                        keep_menu_open = true,
+                    },
+                    {
+                        text = _("Bold"),
```

Comment:
```suggestion
                        text = _("Use bold font"),
```

("Bold" feels a bit short for a menu item. I used "Use monospace font" in the TextEditor plugin - but there, I dunno, it's not like "use any bold font", it's rather "make that font bold")

Reply:
`Bold statusbar`?